fractal_server/app/routes/api/v2/_aux_functions.py

```diff
@@ -362,7 +362,7 @@ async def _get_task_check_owner(
                 ),
             )
         else:
-            owner = user.username or user.slurm_user
+            owner = user.username or user.settings.slurm_user
```

Comment:
make sure you include `verify_user_has_settings`, before calling `user.settings`